Sources/xcproj/PBXProjEncoder.swift

```diff
@@ -10,14 +10,76 @@ extension PlistSerializable {
     var multiline: Bool { return true }
 }
 
+protocol ProjEncoderBuffer {
```

Comment:
Should we make the protocol and the extensions below `fileprivate` since they are only used here?

Reply:
I can try that. I believe they will need to be available for the pbxproj+helpers changes.